templates/templates/_tag_manager.html

```diff
@@ -20,3 +20,5 @@
         '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
         })(window,document,'script','dataLayer','GTM-K92JCQ');</script>
         <!-- End Google Tag Manager -->
+
+        <style>#rememberMe {display: none;}</style>
```

Comment:
I'm not sure what this does, is it intentional?

Reply:
yes, it's a temporary fix for the issue with tag manager that will be fixed by @carkod in some other ticket